### ISSUE TITLE
Application closes abruptly when a user double clicks the application icon

### ISSUE THREAD

**Issue by nus-se-script** [nus-cs2103-AY2425S1/pe-dev-response#606]
**Issue:**
Double clicking the application icon on the top left corner to expand the window causes the system to unexpectedly shut down. (see video attached below)

**Expect:**
The window to maximise

**Actual:**
The application shuts down unexpectedly without warning

**Explanation:**
When users double-click the top-left corner of the application window, expecting to maximize the screen, the application instead closes abruptly, causing confusion and frustration, particularly for new users, since it was never mentioned in the user guide that such a behavior is to be expected. After this unexpected closure, users are left uncertain about whether their work has been saved, leading them to spend time manually verifying each entry to ensure no data is missing. This issue causes significant inconvenience to users who manage large amounts of data, such as numerous contact details, as they may need to spend hours unnecessarily re-checking each entry. This disruptive behavior undermines user confidence and creates a substantial barrier to smooth workflow, making the application feel unreliable and difficult to use.
After careful consideration of this issue's impact, I have assessed it as medium severity. Although a higher severity may be applied due to the disruption to user workflow and potential data risks, I believe that setting it as medium will help us reach a consensus and support our shared goal of enhancing the user experience.

**Operating system:** Windows 11 Home

**Unexpected behavior:** 
<i><video controls><source src="https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/d7be8a4c-1be5-4925-89a9-04bc3f0b4846.mp4" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/d7be8a4c-1be5-4925-89a9-04bc3f0b4846.mp4</i>

<hr>
<sub>[original: nus-cs2103-AY2425S1/pe-interim#858] [original labels: severity.Medium type.FunctionalityBug]</sub>

**Comment by Chronoxy** [comment]
# Team's Response
Thank you for raising this issue.

This feature is a Windows-intended feature. Here are some information regarding this concern that you have, if you would like, enjoy reading.
TLDR; It was a feature in Windows 3.1 (double clicking icon was equivalent to exiting the software; it closes the software rather than crashing the software) that has been inherited to the current windows. Initially they decided to remove it, however, many users wanted it back and therefore it is still present in Windows 11. Therefore, we concur to reject this issue. Thank you

https://learn.microsoft.com/en-au/archive/blogs/jensenh/you-windows-3-1-lovers

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe-dev-response/main/files/ef10fb24-7364-4076-883e-4c9e7ef1297a.png)

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe-dev-response/main/files/03dfb4d5-f542-4540-a03d-23c7b7360abb.png)



## Duplicate status (if any):
--
